### PR TITLE
Arch/bid 70.2

### PR DESCRIPTION
**Pull request by warrenw239**

### CODE REVIEW

File: backend/database/databaseHelpers.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+const DBimports = require('./database');
+
+const createConsumer = (user) => {
```
Comment (shams-ali) [inline backend/database/databaseHelpers.js]: do not merge any code that you arent sure works...

the code in here will not work properly...rememeber....if you write code...you need to obe able too test your code...everything on this page has erroors

File: prettier.config.js [hunk 2]
```diff
@@ -1,6 +1,14 @@
 module.exports = {
+<<<<<<< HEAD
```
Comment (shams-ali) [inline prettier.config.js]: you forgot too review your PR.....

File: src/App.js [hunk 3]
```diff
@@ -1,15 +1,76 @@
...
+                    <Route
+                        exact
+                        path="/ForContractors"
+                        component={ForContractors}
+                    ></Route>
+                </Switch>
+            </Router>
+        );
+    }
+    if (
```
Comment (shams-ali) [inline src/App.js:36]: all this logic should happen in a anoother compoent...ill show you how to do this dont worry about it for now

File: src/auth0-provider-with-history.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+import { Auth0Provider } from '@auth0/auth0-react';
+import PropTypes from 'prop-types';
+
+const Auth0providerWithHistory = ({ children }) => {
```
Comment (shams-ali) [inline src/auth0-provider-with-history.js:6]: i dont see this component being used anywhere

File: src/components/NavBarConsumerHome.js [hunk 5]
```diff
@@ -0,0 +1,77 @@
+import React, { useState } from 'react';
+import '../styles/NavBar.css';
+import LogOutButton from './LogOutButton';
```
Comment (shams-ali) [inline src/components/NavBarConsumerHome.js:2]: your eslint rules should be forcing import organizatino....please add that too youor eslint configuration

File: src/components/NavBarOut.js [hunk 6]
```diff
@@ -1,56 +1,69 @@
 import React, { useState } from 'react';
 import '../styles/NavBar.css';
+import LogInButton from './LogInButton';
```
Comment (shams-ali) [inline src/components/NavBarOut.js:2]: navbar for abouot and navbar out look exactly the same...consider using reusable components

File: src/pages/LoggedInConsumer.js [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+        axios({
+            method: 'post',
+            url: 'http://localhost:3000/data',
+            params: {
+                user: user,
+            },
+        });
+    });
+
+    //NOTE: more to add here. do not change to implicit return
```
Comment (shams-ali) [inline src/pages/LoggedInConsumer.js]: i gave you wrong conventino my bad it shuold be:

`// TODO:`

File: src/styles/LogInButton.css [hunk 8]
```diff
@@ -0,0 +1,7 @@
+.btn-primary {
+    display: flex;
+    margin-top: 20px;
+    height: 30px;
+    background-color: rgb(92, 92, 220);
+    color: white;
+}
```
Comment (shams-ali) [inline src/styles/LogInButton.css]: missing end of line

File: prettier.config.js (hunk 2 above)
Comment (shams-ali) [reply prettier.config.js]: merge coonflicts

File: src/components/NavBarOut.js (hunk 6 above)
Comment (warrenw239) [reply src/components/NavBarOut.js:2]: NavBarForAboutPages has two primary differences. its button on the slider goes back to the landing page, and there is not log in button on the about pages nav bar. all of my nav bars throughout the project will be very similar but have small differences like that. if there is a better way to make it more resuable im fine with it.

File: prettier.config.js (hunk 2 above)
Comment (warrenw239) [reply prettier.config.js]: weird that lint let this slip through. there were 3 others that it pointed our for me when i was commiting, and no file error anywhere was showing up for this.

**Pull request closed without merging**